From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Tue, 28 Apr 2009 19:08:19 +0200
Subject: [PATCH] setup.py: skip import check while cross compiling

during cross compilation we cannot import freshly for the target
compiled modules into the build python. This patch skips this check.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 setup.py |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/setup.py b/setup.py
index d0118ae..d397504 100644
--- a/setup.py
+++ b/setup.py
@@ -292,6 +292,10 @@ class PyBuildExt(build_ext):
             self.announce('WARNING: skipping import check for Cygwin-based "%s"'
                 % ext.name)
             return
+        if os.environ.get('CROSS_COMPILING') == 'yes':
+            self.announce('WARNING: skipping import check for cross compiled "%s"'
+                % ext.name)
+            return
         ext_filename = os.path.join(
             self.build_lib,
             self.get_ext_filename(self.get_ext_fullname(ext.name)))
